app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -63,7 +67,7 @@ const DosesDescriptionList = ({ dhbData }: DosesDescriptionListProps) => (
       hasMetTarget={dhbData.hasMetSecondDoseTarget}
```

Comment:
While we are here did we want to go back to `dose` vs `doses`?  Can be seperate 

otherwise LGTM